README.md

```diff
@@ -1 +1,14 @@
-.
+## Lalamove's Delivery SDK
+
+To install the dependencies run `npm install`
+
+To build the SDK run `npm run build`
+
+To test the SDK locally:
+
+1. Make sure you have built the SDK
+2. Run `mkdir testing` in project root
```

Comment:
Can we also create a Make command (ex: `make dev`) to execute these commands?

Reply:
of course